immuni_exposure_ingestion/models/upload_eu.py

```diff
@@ -48,7 +48,7 @@ def countries_to_process(cls) -> Cursor:
 
         :return: the cursor that iterates over distinct countries that are yet to be processed.
         """
-        return cls.objects.filter(to_publish=True).distinct(field="country")
+        return cls.objects.filter(to_publish=True, country__ne="IT").distinct(field="country")
```

Comment:
I wouldn't hardcode such info within the model. It's more of a "knowledge of the caller".

What about the following?

```python
@classmethod
def countries_to_process(cls, excluded: Optional[Set[str]]) -> Cursor:
    ...
    return cls.objects.filter(to_publish=True, country__nin=excluded).distinct(field="country")
```

(Maybe the driver requires casting to list `country__nin=list(excluded)`, but hopefully not)

Usage:
```python
UploadEU.countries_to_process(exclude={"IT"})
```